### PR TITLE
Deposit & landing page CSS refactor

### PR DESCRIPTION
**Pull request by kpsherva**

### CODE REVIEW

File: invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/EditButton.js [hunk 1]
```diff
@@ -23,17 +22,19 @@ export const EditButton = (props) => {
       })
       .catch((error) => {
         setLoading(false);
-        handleError(error.response.data.message);
+        onError(error.response.data.message);
```
Comment (ntarocco) [inline invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/EditButton.js:25]: may I ask you to change this to `try/catch`?

**Review by ntarocco — APPROVED** [review]
Checked quickly, massive work! Good job!
Remember to change the `rm-` classes. :)

File: invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/files.html [hunk 2]
```diff
@@ -98,13 +98,13 @@
 
 {% macro file_list_box(files, pid, is_preview, record) %}
 <div class="">
-  <div class="ui accordion panel {{record.ui.access_status.id}}" id="preview" href="#collapsablePreview">
-    <div class="active title panel-heading {{record.ui.access_status.id}}" tabindex="0">
+  <div class="ui accordion panel mb-10{{record.ui.access_status.id}}" id="preview" href="#collapsablePreview">
```
Comment (jennur) [inline invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/files.html:101]:
```suggestion
  <div class="ui accordion panel mb-10 {{record.ui.access_status.id}}" id="preview" href="#collapsablePreview">
```

File: invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordVersionsList.js [hunk 3]
```diff
@@ -38,14 +45,14 @@ const RecordVersionItem = ({ item, activeVersion }) => {
           }
 
           {doi && (
-            <small className={'doi' + (activeVersion ? ' text-muted-on-bg' : ' text-muted')}>
+            <small className={'doi' + (activeVersion ? ' .text-muted-darken' : ' text-muted')}>
```
Comment (jennur) [inline invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordVersionsList.js:48]:
```suggestion
            <small className={'doi' + (activeVersion ? ' text-muted-darken' : ' text-muted')}>
```

File: invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordVersionsList.js [hunk 4]
```diff
@@ -38,14 +45,14 @@ const RecordVersionItem = ({ item, activeVersion }) => {
...
               {doi}
             </small>
           )}
         </List.Content>
 
         <List.Content floated="right">
-          <small className={activeVersion ? 'text-muted-on-bg' : 'text-muted'}>
+          <small className={activeVersion ? '.text-muted-darken' : 'text-muted'}>
```
Comment (jennur) [inline invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordVersionsList.js:55]:
```suggestion
          <small className={activeVersion ? ' text-muted-darken' : 'text-muted'}>
```

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides [hunk 5]
```diff
@@ -227,14 +227,181 @@ pre.export.result {
...
+  display: block;
+  border-style: none;
+}
+
+dd {
+  // Start left aligned
+  margin-inline-start: 0;
+}
+
+dl.details-list {
```
Comment (jennur) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides:267]: Should this be in `list.overrides`?

**Review by jennur — APPROVED** [review]

**Comment by zzacharo** [comment]
continued in https://github.com/inveniosoftware/invenio-app-rdm/pull/1396

**Comment by zzacharo** [comment]
continued in https://github.com/inveniosoftware/invenio-app-rdm/pull/1396

**Pull request closed without merging**